src/preferences/dialog/dlgprefsound.cpp

```diff
@@ -370,6 +412,22 @@ void DlgPrefSound::loadSettings(const SoundManagerConfig &config) {
         deviceSyncComboBox->setCurrentIndex(0);
     }
 
+    if (m_config.getAPI() == MIXXX_PORTAUDIO_JACK_STRING) {
+        // TODO(Be): Get the processing latency from JACK. PortAudio does
+        // not support this as of June 2017.
+        roundTripLatencySpinBox->setMinimum(0.0);
+    } else {
+        bool measuredRoundTripLatencyAtDefault =
+            m_pRoundTripLatency->get() == roundTripLatencySpinBox->minimum();
+        roundTripLatencySpinBox->setMinimum(config.getProcessingLatency() * 2.0);
```

Comment:
Testing with JACK would be tricky with your setup because you use two sound cards. So try commenting out this line. Then use a buffer size that results in a round trip latency around 30-40 ms. With the MASTER or MASTER_AND_BOOTH mode an the mic TALK button enabled, play a track with a steady beat and count along. Try switching back and forth between setting the Measured Round Trip Latency to 0 and setting it to what you measured.